lib/matplotlib/axis.py

```diff
@@ -2004,7 +2004,7 @@ def get_tick_space(self):
             # is no more than 3:1
             size = tick.label1.get_size() * 3
             size *= np.cos(np.deg2rad(tick.label1.get_rotation()))
-            self._tick_space = np.floor(length / size)
+            return np.floor(length / size)
```

Comment:
So we're no longer caching the calculation?  This could be somewhat expensive to do every time.  If not too expensive, we should remove the `if self._tick_space` (and the self._tick_space member) altogether.  If it is, maybe we just need to properly invalidate this value using the `stale` infrastructure (which will be traitlets sometime in the near future)?